### PR TITLE
OSGi compliant jars

### LINKED ISSUE #59: OSGi compliant jars
Are there any plans to make the Sprotty jars OSGi compliant? Since I currently use them within the Eclipse ecosystem I had to create a separate project just to wrap the jars, which is not ideal.

### PR DESCRIPTION
**Pull request by tomvdbussche**
Fixes #59

### CODE REVIEW

**Comment by tomvdbussche** [comment]
Based on https://github.com/eclipse/lsp4j/blob/master/gradle/manifest-gen.gradle with minor tweaks to make use of guava provided by orbit. Tested locally and it seems to work correctly.

**Comment by tomvdbussche** [comment]
Also I didn't know if I needed to change the license header since I basically duplicated it from the lsp4j project.

File: gradle/manifest-gen.gradle [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+if (baseVersion.endsWith('-SNAPSHOT'))
+	baseVersion = baseVersion.substring(0, baseVersion.length() - 9)
+else if (baseVersion.split('\\.').length == 4)
+	baseVersion = baseVersion.substring(0, baseVersion.lastIndexOf('.'))
+def qualifiedVersion = baseVersion + '.v' + buildTime
+
+jar.manifest {
+	version = qualifiedVersion
+	if (project.hasProperty('title'))
+		name = project.title
```
Comment (spoenemann) [inline gradle/manifest-gen.gradle]: There seems to be a bug in the build config from LSP4J. Please remove lines 24 and 25 and add this block after the `jar.manifest` block instead:
```groovy
afterEvaluate {
	if (project.hasProperty('title'))
		jar.manifest.name = project.title
}
```

Comment (tomvdbussche) [reply gradle/manifest-gen.gradle]: Ok that does seem to do the trick!

**Comment by spoenemann** [comment]
Thanks!

**Pull request merged**